module/ui.go

```diff
@@ -0,0 +1,11 @@
+package module
+
+import (
+	"github.com/GoCodeAlone/modular"
+	"github.com/GoCodeAlone/workflow/ui"
+)
+
+// NewUIModule creates a new UI module
+func NewUIModule(name string, config map[string]interface{}) modular.Module {
+	return ui.NewUIModule(name, config)
```

Comment:
Is there any point for this file to even exist? All it's doing is re-exposing ui.NewUIModule. There's no point for this file to exist, just reference the target module using it's correct path.